configs/polarRegions.conf

```diff
@@ -0,0 +1,143 @@
+[climatologyMapWoa]
+## options related to plotting climatology maps of Temperature and Salinity
+## fields at various levels, including the sea floor against control model
+## results and WOA climatological data
+
+# list of depths in meters (positive up) at which to analyze, 'top' for the
+# sea surface. Note that, for seasons='ANN', depths can be as deep as 5500 m,
+# otherwise use a maximum depth of 1500 m.
+depths = ['top', -50, -200, -400, -600, -800]
+
+[climatologyMapWoaTemperature]
+## options related to plotting climatology maps of potential temperature
+## at various levels, including the sea floor against control model results
+## and WOA18 climatological data
+
+# A dictionary with keywords for the norm
+normArgsResult = {'vmin': -2., 'vmax': 2.}
+
+[climatologyMapWoaSalinity]
+## options related to plotting climatology maps of salinity
+## at various levels, including the sea floor against control model results
+## and WOA18 climatological data
+
+# A dictionary with keywords for the norm
+normArgsResult = {'vmin': 33.8, 'vmax': 35.0}
+
+
+[TSDiagramsForAntarcticRegions]
+## options related to plotting T/S diagrams of Antarctic regions
+
+# The minimum and maximum depth over which fields are plotted, default is
+# to take these values from the geojson feature's zmin and zmax properties.
+zmin = -1000
+zmax = 0
+
+
+[soseTransects]
+## options related to plotting model vs. Southern Ocean State Estimate (SOSE)
+## transects.
+
+# longitudes of transects
+# Western Weddell 1, Filchner (318=42W), Western Weddell 2 (325=35W),
+# West Maud Rise, Fimbul (0), Amery (75E), Totten (117E),
+# George V coast, Merzt (145E), Wilkes (160E), Western Ross (184=176W),
+# Center Ross (187=173W), Eastern Ross (198=162W),
+# Amudsen, Thwaites (253=107W), Bellingshausen (280=80W),
+# West Antarctica (288=72W)
+# Note: Transects at XXX, XXX, XXX, XXX and XXX are chosen for comparison with
+#       Whitworth et al. (XXXX) doi: XXXX/XXXX
```

Comment:
@milenaveneziani, could you edit this to have the right info here?  I think you can just edit the file directly on GitHub if you want to.  You can even use the "insert a suggestion" right here but I think it only edits one line at a time.

Reply:
really? I didn't know I could do that. Let me  try it.